include/swift/Basic/Platform.h

```diff
@@ -22,6 +22,17 @@ namespace llvm {
 }
 
 namespace swift {
+
+  enum class DarwinPlatformKind : unsigned {
```

Comment:
Can I say "please change these names in Clang"? "MacOS" in particular is incredibly wrong.